### ISSUE TITLE
bug: The QuickButton component floats over the mobile/tablet drop down navbar menu

### ISSUE THREAD

**Issue by debsouryadatta** [CircuitVerse/cv-frontend-vue#231]
**Describe the bug**
In Mobile or tablets when the navbar is shown as a dropdown menu, the QuickButton component floats over that drop down menu which may resist the user in accessing the navbar links.

**To Reproduce**
1. Go to [cv-frontend-vue](https://circuitverse.netlify.app/).
2. Open as phone or tablet view.
3. Click on the navbar drop down menu.
4. The bug will be visible.

**Expected behavior**
The QuickButton component should remain below the navbar dropdown menu.

**Screenshots**
![Screenshot from 2023-12-28 00-49-51](https://github.com/CircuitVerse/cv-frontend-vue/assets/91617309/ce305963-4bcf-419c-95d3-ee479bce62a3)

**Device Information [optional]:**
- OS: Ubuntu
- Browser: Chrome
- Version: Chrome 120.0.6099.129

**Additional context**
NA

**Are you working on this issue? (Yes/No)**
Yes

**Comment by pankaj-2503** [comment]
I don't think this is a bug as the user can click the drop down menu to close it so he can work on designing the circuits .Close this issue.
